src/main/xcomponents/styledComponents/Button.jsx

```diff
@@ -9,11 +9,7 @@ const Button = styled.button`
     return '#1F4E7A';
   }};
   border: none;
-  color: ${(props) => {
```

Comment:
I believe this styledComponent prop conditional is used elsewhere. If you absolutely need your `color` to be white, add another condition.

Reply:
I ceated styledComponents2 directory so as to not change the styles currently being used by RequestBar.js